auth/google.go

```diff
@@ -18,10 +18,8 @@ import (
 	"zuri.chat/zccore/utils"
 )
 
-// var ClientId string = "107847729811-5gpmlkee2hm6urdlvm90l47nke6cg0og.apps.googleusercontent.com" // localhost:8080
-// var ClientSecret string = "gE5Si1htrsoPKOIUIeDaNZIn"                                             // localhost:8080
-var ClientId string = "107847729811-bc61afgl2s8tcqqae02dn67l27kue16o.apps.googleusercontent.com" // api.zuri.chat
-var ClientSecret string = "Zq972KMNCGA_pBkeHEZafpm2"                                             // api.zuri.chat
+var ClientId string = "107847729811-1i98f99fhamo3rg9930430tsua2he1ig.apps.googleusercontent.com" // api.zuri.chat
+var ClientSecret string = "W1sRuhVfIJzdJgO_Ryyj-pkQ"                                             // api.zuri.chat
```

Comment:
Why not move this to Config?

Reply:
I am still testing this, I have changed the values and will keep changing it, leave it alone for now